operations/customers.md

```diff
@@ -505,9 +522,17 @@ Updates personal information of a customer. Note that if any of the fields is le
 | `Visa` | [Document](customers.md#document) | optional | New visa details. |
 | `DriversLicense` | [Document](customers.md#document) | optional | New drivers license details. |
 | `Address` | [Address parameters](customers.md#address-parameters)  | optional | New address details. |
+| `LegalIdentifiers` | [Legal identifiers update](#legal-identifiers) | optional | Update to legal identifiers. |
 | `Classifications` | array of [Customer classification](customers.md#customer-classification) | optional | New classifications of the customer. |
 | `Options` | array of [Customer option](customers.md#customer-option) | optional | Options of the customer. |
 
+#### Legal identifiers update
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ItDestinationCode` | [String update value](reservations.md#string-update-value) | optional | Update to Italian destination code. |
```

Comment:
Everywhere else, we use format `New {property Name}.` for description of update value. Unless there is reason for this new format, I'd stick to the standard that's used everywhere else in the documentation.